README.md

```diff
@@ -43,6 +44,13 @@ func main() {
 }
 ```
 
+2. This file may be used to test any recent local changes because
+   `github.com/heimweh/go-pagerduty` is the referenced module in `go.mod`
+3. So, in the project root run:
+```bash
+$ TF_PAGERDUTY_CACHE=memory PAGERDUTY_TOKEN=<SECRET> go run <PATH/TO>/main.go
```

Comment:
`TF_PAGERDUTY_CACHE` environment variable is optional a more suitable for real world scenarios which need of this performance improvement. Like it's mentioned [here](https://github.com/heimweh/go-pagerduty#caching-support).

Reply:
Sounds good, I'll leave this out since it is now better explained in the README.